pyvista/plotting/renderer.py

```diff
@@ -1267,6 +1276,17 @@ def show_bounds(
             raise ValueError(f'padding ({padding}) not understood. Must be float between 0 and 1')
         cube_axes_actor.SetBounds(bounds)
 
+        # update the values of range of each axis
+        if isinstance(axes_range, (collections.abc.Sequence, np.ndarray)):
+            if len(axes_range) == 6:
+                cube_axes_actor.SetXAxisRange(axes_range[0], axes_range[1])
+                cube_axes_actor.SetYAxisRange(axes_range[2], axes_range[3])
+                cube_axes_actor.SetZAxisRange(axes_range[4], axes_range[5])
+            else:
+                raise ValueError(
+                    'axes_range must be passed as a [xmin, xmax, ymin, ymax, zmin, zmax] list or tuple'
```

Comment:
Depending on how you read it we might need "an" here: is it "_an xmin xmax..._" or is it "_a bracket xmin xmax ..._"?
```suggestion
                    'axes_range must be passed as an [xmin, xmax, ymin, ymax, zmin, zmax] list or tuple'
```